test/TestIncludeFromVanillaProj/TestIncludeFromVanillaProj.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+      <ProjectReference Include="../TestProjectWithWarnings/TestProjectWithWarnings.csproj">
+        <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
```

Comment:
I believe we can have the `ReferenceOutputAssembly` as an attribute on the `ProjectReference` element as well which is a bit cleaner IMHO.